controllers/user.js

```diff
@@ -62,7 +62,7 @@ class UserController {
             const { username, password } = req.body;
             const actualUser = await User.findOne({
                 where: { username },
-                include: [UserProfile, Bans]
+                include: [UserProfile, Ban]
```

Comment:
change Ban to userBan table
if got error on running; remove include and put this code

const userBanned = await userBans.findOne ({where: {userID: actualUser.id} })
if(!userBanned){
//throw BANNED error
}